README.md

```diff
@@ -75,6 +75,11 @@ userPassword: password
 
 3- Build with `docker-compose build` (it's easier to use the command line than Intellij)
 
+Since docker image /unicon/shibboleth-sp does not exist anymore in the Docker official repository. You will have to
+copy the image from someone's else machine and load it into your Docker. 
```

Comment:
```suggestion
copy the image from someone else's machine and load it into your Docker. 
```

PS: Is the image file too large? Could we add it into the repo ?

Reply:
We may actually need to update this to use the https://hub.docker.com/r/unicon/shibboleth-idp image now. But this will require a bit of work, should be a dedicated technical task ticket. Can you make sure it's created and talk about it to prioritize this with the BCM?